rna-seq-pipeline.wdl

```diff
@@ -55,6 +55,9 @@ workflow rna {
         File rna_qc_tr_id_to_gene_type_tsv
         String? mad_qc_disk
         String? rna_qc_disk
+        # These are for internal use, leave undefined
+        Int? kallisto_fragment_length_undefined
+        Float? kallisto_sd_undefined
```

Comment:
If these are for internal use then you should put these outside of the `input` block so users can't accidentally specify them.

Reply:
I think it may not work for some reason. Let me try..

Reply:
Yeah, one is not allowed to declare optional variables.

Reply:
Would it work to specify it as a normal variable? Like this:
```
Int kallisto_fragment_length_undefined = -1
```

Reply:
No. This is a kind of an unholy workaround around the lack of `null` number value in wdl..